lib/src/policy/attributes.ts

```diff
@@ -89,6 +89,10 @@ export type Attribute = {
   metadata?: Metadata;
 };
 
+// This is not currently needed by the client, but may be returned.
+// Setting it to unknown to allow it to be ignored for now.
+export type SubjectMapping = unknown;
```

Comment:
@jakedoublev is there an option to not return Subject Mappings ?

Reply:
I think [they are always returned](https://github.com/opentdf/platform/blob/b7d694d5df3867f278007660c32acb72c868735e/service/policy/db/query.sql#L146) after the migration in policy to `sqlc`. Mind filing an issue if that is a behavior that is a concern?

Reply:
Filed https://github.com/opentdf/platform/issues/1559